libinitializer/SecureInitializer.cpp

```diff
@@ -208,3 +209,343 @@ std::shared_ptr<bas::context> SecureInitializer::SSLContext(Usage _usage)
     // if not found, return default
     return defaultP->second;
 }
+
+struct ConfigResult
+{
+    KeyPair keyPair;
+    std::shared_ptr<boost::asio::ssl::context> sslContext;
+};
+
+ConfigResult initOriginConfig()
+{
+    std::string dataPath = "conf/";
```

Comment:
config path get from config file maybe better, I will modify this in future

Reply:
thanks